extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventClass.schema.json

```diff
@@ -17,7 +17,13 @@
   "description": "A time-series based Journey Step Event Class.",
   "definitions": {
     "journeyStepEventClass": {
-      "properties": {}
+      "properties": {
```

Comment:
Do we specifically need a class here ?

Reply:
It is an already existing class, re-using the same for storing the serviceType like, 
"serviceType" : "serviceEvents.segmentExportJob" OR "serviceType" : "serviceEvents.dispatcher"